docs/bigquery/snippets.py

```diff
@@ -495,53 +495,140 @@ def do_something(row):
 
 
 def test_load_table_from_uri(client, to_delete):
-    ROWS = [
-        ('Phred Phlyntstone', 32),
-        ('Bharney Rhubble', 33),
-        ('Wylma Phlyntstone', 29),
-        ('Bhettye Rhubble', 27),
+    dataset_id = 'load_table_dataset_{}'.format(_millis())
+    dataset = bigquery.Dataset(client.dataset(dataset_id))
+    client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    # [START bigquery_load_table_gcs_json]
+    # dataset_id = 'my_dataset'
+    dataset_ref = client.dataset(dataset_id)
+    job_config = bigquery.LoadJobConfig()
+    job_config.schema = [
+        bigquery.SchemaField('name', 'STRING'),
+        bigquery.SchemaField('post_abbr', 'STRING')
     ]
-    HEADER_ROW = ('Full Name', 'Age')
-    bucket_name = 'gs_bq_load_test_{}'.format(_millis())
-    blob_name = 'person_ages.csv'
-    bucket, blob = _write_csv_to_storage(
-        bucket_name, blob_name, HEADER_ROW, ROWS)
-    to_delete.extend((blob, bucket))
-    DATASET_ID = 'delete_table_dataset_{}'.format(_millis())
-    dataset = bigquery.Dataset(client.dataset(DATASET_ID))
+    job_config.source_format = 'NEWLINE_DELIMITED_JSON'
```

Comment:
Do we not have constants for things like this?

Reply:
Not that I could find in https://googlecloudplatform.github.io/google-cloud-python/latest/bigquery/reference.html Filed https://github.com/GoogleCloudPlatform/google-cloud-python/issues/4793